feedback-module/src/components/pages/Screen2.js

```diff
@@ -8,34 +8,66 @@ import ModuleButton from "../common/Button";
 
 function Screen2({ feedbackType }) {
   const [checkedFields, setCheckedFields] = useState({});
```

Comment:
Since we're storing the checkedFields in objects and using the labels as keys, I'm wondering how this will affect us in the future when we're parsing the data to be sent back to the webmasters. Because different forms will have different labels & thus different keys, it seems like it would be more challenging to do things like filtering the data or formatting.

Reply:
Interesting I could see how this might present challenges but how would we format it in an array because don't we want to know which checkboxes have been checked and which haven't (this is the question I was asking you about yesterday morning when I was setting up the data structure)

Reply:
I was thinking an object could work because we could just send this object to the backend and it will be very clear what questions were asked and what the responses were (the recipient of the email doesn't need to know the questions that the user didn't see I don't think)

Reply:
I was thinking that instead of the array containing just the checked elements, we can map through the checkbox labels and create an array of objects with the format `{label: "checkbox label", checked: false}`. Then for the `onCheck` function, instead of passing in the label, we can pass in the index.